### ISSUE TITLE
Question about the effectiveness of Chinese speech quality evaluation

### ISSUE THREAD

**Issue by xiaoyiming** [gabrielmittag/NISQA#9]
Hi:
Thank you for your hard work, I have a  question, would be great if you could find some time to answer it.
The pre-trained model "NISQA (v2.0) mos only"  you provided shows good results on the English data-set. However, performance results are limited on our self-made Chinese data-set. We collected 2000  simulated distortions speech and obtained their MOS by crowdsourcing scoring.  However, The PCC of "NISQA (v2.0) mos only" is only 68% and the RMSE is 0.99. 
Does the provided pre-training model have greater restrictions on the type of speech?
Thanks
Yimingxiao

**Comment by gabrielmittag** [comment]
Hi,

That is interesting, the pre-trained model was actually also trained with Chinese speech samples, although the majority was English and other European languages so there could be an effect of the language. What kind of distortions has your Chinese dataset? Are they different to the distortions of the English dataset? 

The main focus of the model are distortions caused by communication networks, such as codecs, background noises, packet loss. It is not focused on noise suppression / speech enhancment.

Best,
Gabriel

**Comment by xiaoyiming** [comment]
Thanks for your reply. 
The distortions  we considered  are similar to your paper. Including codecs, packet loss, background noise, etc. Of course, our Chinese speech samples  also consider the case of speech enhancement.  Because the current popular communication systems have speech enhancement modules. Will it affect the results?

**Comment by gabrielmittag** [comment]
Hm, it could be either the speech enhancement conditions or it could be due to the language. You could finetune the model with your own dataset with a smaller learning rate just for a few epochs, the model should pick up the new conditions quickly.

**Comment by xiaoyiming** [comment]
Thanks for your advice!
